content/docs/operations/debug.md

```diff
@@ -5,46 +5,41 @@ weight: 15
-
-## Before you begin
-
-If you have not done yet, install the `{{< reuse "docs/snippets/cli-name.md" >}}` CLI. The `{{< reuse "docs/snippets/cli-name.md" >}}` CLI is a convenient tool that helps you gather important information about your gateway proxy. To install the `{{< reuse "docs/snippets/cli-name.md" >}}`, you run the following command: 
-```sh
-curl -sL https://run.solo.io/gloo/install | sh
-export PATH=$HOME/.gloo/bin:$PATH
-```
-
-{{% callout type="info" %}}
-Make sure to use the version of `{{< reuse "docs/snippets/cli-name.md" >}}` that matches your installed version.
-{{% /callout %}}
+{{< reuse "docs/snippets/product-name-caps.md" >}} is based on [Envoy proxy](https://www.envoyproxy.io). If you experience issues in your environment, such as policies that are not applied or traffic that is not routed correctly, in a lot of cases, these errors can be observed at the proxy. In this guide you learn how to use the {{< reuse "docs/snippets/product-name.md" >}} and Envoy debugging tools to troubleshoot misconfigurations on the gateway.
 
 ## Debug your gateway setup
 
-1. Check the {{< reuse "docs/snippets/product-name.md" >}} installation. You can do that by using the `{{< reuse "docs/snippets/cli-name.md" >}} check` [command](/docs/reference/cli/glooctl_check/) that quickly checks the health of {{< reuse "docs/snippets/product-name.md" >}} deployments, pods, and custom resources, and verifies Gloo resource configuration. Any issues that are found are reported back in the CLI output. 
+1. Make sure that the {{< reuse "docs/snippets/product-name.md" >}} control plane and gateway proxies are running.
+   
+   ```shell
+   kubectl get pods -n {{< reuse "docs/snippets/ns-system.md" >}}
+   ```
+   <!-- TODO: CLI You can do that by using the `{{< reuse "docs/snippets/cli-name.md" >}} check` [command](/docs/reference/cli/glooctl_check/) that quickly checks the health of {{< reuse "docs/snippets/product-name.md" >}} deployments, pods, and custom resources, and verifies Gloo resource configuration. Any issues that are found are reported back in the CLI output. 
    ```sh
    {{< reuse "docs/snippets/cli-name.md" >}} check
    ```
    
    Example output for a misconfigured VirtualHostOption:
    ```console
-   Found rejected VirtualHostOption by 'gloo-system': gloo-system jwt (Reason: 2 errors occurred:
+   Found rejected VirtualHostOption by '{{< reuse "docs/snippets/ns-system.md" >}}': {{< reuse "docs/snippets/ns-system.md" >}} jwt (Reason: 2 errors occurred:
 	* invalid virtual host [http~bookinfo_example] while processing plugin enterprise_warning: Could not load configuration for the following Enterprise features: [jwt]
    ```
-   
-2. Get the details for the failed resource. For example to get the details of a VirtualHostOption, you can use the following command. 
+   -->
+2. Get the details for any failed resource. For example to get the details of an HTTPRoute, you can use the following command. 
```

Comment:
this step does not fit to step 1 anymore. I am wondering if we should tell people to check the status of HTTPRoutes and all policies to see if there is an error.